project.clj

```diff
@@ -181,7 +181,20 @@
   ; use `lein deps :plugins-tree` to inspect conflicts
   :plugins [[lein-shell "0.5.0"]
             [org.clojure/clojure ~clj-version] ;override perforate
-            [perforate ~perforate-version]]
+            [perforate ~perforate-version]
+            [reifyhealth/lein-git-down "0.3.5"]]
+  :middleware [lein-git-down.plugin/inject-properties]
+  :repositories [["public-github" {:url "git://github.com"}]
+                 ["private-github" {:url "git://github.com" :protocol :ssh}]]
+  ; to simultaneously work on an upstream dependency and have
+  ; Travis pick up on it:
+  ; 1. add an entry mapping the upstream's maven coordinate to its dev GitHub repository
+  ;    in the :git-down map:
+  ;    eg., To work on ctim in the `frenchy64` fork:
+  ;         :git-down {threatgrid/ctim {:coordinates frenchy64/ctim}}
+  ; 2. change the upstream dependency's version to the relevant sha
+  ;    eg., [threatgrid/ctim "9acbc93333d630d9b9a0a9fc19981b0ba0ddec1c"]
+  ;:git-down {threatgrid/ctim {:coordinates frenchy64/ctim}}
```

Comment:
I like this! Much more cleaner than the deployment of snapshot versions that can be overwritten between multiple branches.

Reply:
Yes :-)!

Reply:
indeed that's awesome, kind of melpa for lein